app.py

```diff
@@ -19,8 +19,11 @@
 
 import logging
 
+from typing import Optional
+
 import click
 import thoth.storages.sync as thoth_sync_module
+from thoth.storages.sync import _HANDLERS_MAPPING
```

Comment:
We might want to make this public so we do not use private API outside of the package.

Reply:
https://github.com/thoth-station/storages/pull/2125